### ISSUE TITLE
Empty expense_documents on analyze_expense

### ISSUE THREAD

**Issue by arsher-b** [aws-samples/amazon-textract-textractor#370]
The analyze_expense return 0 on `expense_documents` even the image is a receipt. 
Here's the receipt used:
![2405260266_2](https://github.com/aws-samples/amazon-textract-textractor/assets/129343886/f7143f39-1ad5-4076-9a45-8bfeb7057235)

![image](https://github.com/aws-samples/amazon-textract-textractor/assets/129343886/d32ccc57-5b07-48dc-b99e-0cc4f9ce79ad)

Output:
![image](https://github.com/aws-samples/amazon-textract-textractor/assets/129343886/27e5d583-7978-44a6-aa0f-d9a345f3fb3a)

Can someone fix this where it still parse the receipt as line items inside the `Expense Documents` instead of `Lines`?

**Comment by athewsey** [comment]
Verified I can reproduce this - the document works in the Textract console and the API response contains a non-empty `ExpenseDocuments`.

The problem appears to be [this line](https://github.com/aws-samples/amazon-textract-textractor/blob/e830731c191b4de30f684876020c62e74caea53b/textractor/parsers/response_parser.py#L1481) (with a `FIXME`), which **specifically ignores** any `ExpenseDocuments` entry with no `SummaryFields`: The example doc contains no summary fields, only line items.

Presumably, this is because Textractor needs to [determine the page number](https://github.com/aws-samples/amazon-textract-textractor/blob/e830731c191b4de30f684876020c62e74caea53b/textractor/parsers/response_parser.py#L1483) of the expense document immediately after (even though the Textract response API structure does not specifically tie an ExpenseDocument to one Page, and appears to be designed to support multi-page invoices?). Currently it does this by looking for a `PageNumber` annotation on the first summary field.

As a minimally-impacting fix, I think it should also be possible to try fetching this information via:

```python
textract_json["ExpenseDocuments"][0]["LineItemGroups"][0]["LineItems"][0]["LineItemExpenseFields"][0]["PageNumber"]
```

...But may also be worth considering whether Textractor should continue framing ExpenseDocuments as within a single page, rather than potentially spanning multiple pages?

**Comment by athewsey** [comment]
Hi @arsher-b, we believe this should now be solved with the [v1.8.0 release](https://github.com/aws-samples/amazon-textract-textractor/releases/tag/v1.8.0). It'd be great if you could help confirm?

**Comment by arsher-b** [comment]
Hello Sir @athewsey,
I've confirmed that the issue is resolved with the v1.8.0 release. Thanks!
